mxcube3/ui/reducers/sampleview.js

```diff
@@ -7,7 +7,7 @@ const initialState = {
   width: 0,
   height: 0,
   lightOn: false,
-  motors: { Light: {}, Phi: {}, PhiY: {}, PhiZ: {}, Sampx: {}, Sampy: {}, Zoom: {} },
+  motors: {},
```

Comment:
This happens if "http://w-v-kitslab-mxcube-1:8081/mxcube/api/v0.1/sampleview" don't return all the necessary values. 

I think it is dangerous to implement dummy values as this does not expose that the server is not returning all necessary values.